Source/Core/Duality/Cloning/Surrogates/HashSetSurrogate.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Duality.Cloning.Surrogates
+{
+	public class HashSetSurrogate : CloneSurrogate<dynamic>
```

Comment:
Since `dynamic` is a C# compiler alias, not an actual runtime type, please change this to `CloneSurrogate<object>` and use a local `dynamic` variable in the implementation to which the `object` parameter is assigned.